Mueez Afzal-weatherman.py

```diff
@@ -0,0 +1,304 @@
+import os
+months={'1':'January', '2':'February','3':'March', '4':'April', '5':'May', '6':'June', '7':'July','8':'August', '9':'September','10':'October','11': 'November','12': 'December'}
+
+
+class year:

```

Comment:
all this should be calculated and shown on console on run time after reading appropriate data given by the commandline we don't need to store results.